src/collective/volto/formsupport/restapi/deserializer/blocks.py

```diff
@@ -1,12 +1,34 @@
+from plone.api.portal import get_registry_record
+from plone.api.portal import set_registry_record
 from plone.dexterity.interfaces import IDexterityContent
 from plone.restapi.bbb import IPloneSiteRoot
 from plone.restapi.interfaces import IBlockFieldDeserializationTransformer
 from Products.PortalTransforms.transforms.safe_html import SafeHTML
+from uuid import uuid4
 from zope.component import adapter
 from zope.interface import implementer
 from zope.publisher.interfaces.browser import IBrowserRequest
 
 
+GLOBAL_FORM_REGISTRY_RECORD_ID = (
```

Comment:
could be a good idea moving this constant in a common place and import it also in post adapter?